napari/settings/_tests/test_shortcuts.py

```diff
@@ -0,0 +1,232 @@
+from typing import Dict, List
+from unittest.mock import Mock
+
+import pytest
+from app_model.types import KeyBinding, KeyCode, KeyMod
+
+from napari.settings._shortcuts import ShortcutRule, ShortcutsSettings
+
+
+def test_shortcut_settings_load_legacy():
+    _legacy_shortcuts = {
+        # viewer
+        'napari:toggle_console_visibility': [
+            KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.KeyC
+        ],
+        'napari:reset_scroll_progress': [KeyCode.Ctrl],
+        'napari:toggle_ndisplay': [KeyMod.CtrlCmd | KeyCode.KeyY],
+        'napari:toggle_theme': [KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.KeyT],
+        'napari:reset_view': [KeyMod.CtrlCmd | KeyCode.KeyR],
+        'napari:delete_selected_layers': [KeyMod.CtrlCmd | KeyCode.Delete],
+        'napari:show_shortcuts': [KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.Slash],
```

Comment:
Such constants should be defined outside the test (global constant or fixture). It reduces readability.